docs/data/horizon/api-reference/structure/README.mdx

```diff
@@ -5,4 +5,8 @@ sidebar_label: Structure
 description: Horizon is an API for interacting with the Stellar network.
 ---
 
+import DocCardList from "@theme/DocCardList";
```

Comment:
Our current [Horizon API Reference page](https://developers.stellar.org/docs/data/horizon/api-reference/structure) is missing cards so I added cards since it has submenus